src/components/BorrowerProfile/SupporterDetails.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<div>
+		<!-- eslint-disable-next-line max-len -->
+		<div class="tw-block md:tw-inline-block md:tw-flex-none md:tw-mr-2 tw-w-[135px] md:tw-w-[150px] lg:tw-w-[84px] xl:tw-w-[96px]">
+			<borrower-image
```

Comment:
@emuvente Trying reuse this component here was my suggestion which is essentially working though it brings in some slightly different image handling than the design with the letterboxed scaled image. I'm ok with the differences for this PR and we can discuss options to either enhance this component or do something else to get things closer to the design later.